website/docusaurus.config.js

```diff
@@ -111,6 +111,11 @@ const config = {
             position: "left",
             label: "Specification",
           },
+          {
+            href: "https://foxglove.dev",
+            label: "Foxglove",
+            position: "right",
```

Comment:
This CTA doesn't seem very clear about why one would want to click on Foxglove from the mcap site?

Reply:
+1